src/main/java/com/google/finapp/SpannerDaoInterface.java

```diff
@@ -0,0 +1,75 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.finapp;
+
+import com.google.cloud.ByteArray;
+import com.google.cloud.spanner.SpannerException;
+
+import java.math.BigDecimal;
+
+/**
+ * The SpannerDaoInterface declares the methods to be used by its separate implementations.
+ */
+public interface SpannerDaoInterface {
+
+  /**
+   * Inserts a new row to the Customer table in the database.
+   *
+   * @param customerId unique the Customer id
```

Comment:
If you don't have any interesting details to add here, you can omit these @params and @throws clauses.